File: Programmers/Prob12901.java
import java.util.Calendar;

class Prob12901 {
    public String solution(int a, int b) {
        String[] answer = { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" };
        Calendar cal = Calendar.getInstance();
        cal.set(2016, a - 1, b);
        return answer[cal.get(Calendar.DAY_OF_WEEK) - 1];
    }

    public static void main(String[] args) {
        int a = 5;
        int b = 24;
        System.out.println(new Prob12901().solution(a, b));
    }
}

File: Programmers/Problem10000s/Prob12901.py
def solution(a, b):
    days_in_month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    days_in_week = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    a -= 1
    b -= 1
    answer = 0
    for i in range(a):
        answer += days_in_month[i]
    answer += b
    answer = days_in_week[answer % 7]
    return answer


a = 5
b = 24
print(solution(a, b))
